target/linux/mediatek/mt7623/config-5.10

```diff
@@ -260,15 +248,14 @@ CONFIG_HARDIRQS_SW_RESEND=y
 CONFIG_HAS_DMA=y
 CONFIG_HAS_IOMEM=y
 CONFIG_HAS_IOPORT_MAP=y
-CONFIG_HAVE_ARM_ARCH_TIMER=y
```

Comment:
There has been some edit war about this one in the past, see 8cdc356f8c30b55698d5f57abbd1f3e16d8c653e